### PR TITLE
Updated a known issue reported by customers

### PR DESCRIPTION
**Pull request by anwara-ms**
Updated a known issue reported by customers + added my name as an author as I look after information barrier feature from security and compliance team. @LolaJacobsen  FYI

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/information-barriers-in-teams.md [hunk 1]
```diff
@@ -173,6 +174,9 @@ Click [here](https://docs.microsoft.com/sharepoint/information-barriers#segments
 
 For more details, including plans and pricing, see [Licensing Guidance](https://docs.microsoft.com/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance).
 
+## Known Issues
+- **Users cant join meetings** : If IB policies are enabled, users are not allowed to join meetings IF meeting roster size is more than 250 users. Root cause is that we IB checks rely on whether users can be added to meeting chat roster and take that signal to allow user to join meeting. Joining a meeting once will add that user to roster, hence for recurring meetings, the roster fills up fast. Once it reaches to a count of 250 users, no additional users are allowed to be added to meeting chat roster. If IB is enables, users are not allowed to join meeting, but if IB is not enabled, users are allowed to join meeting, though they wont be added to meeting chat roster. Short term solution is to remove inactive members from meeting chat roster to make space for new users. Later, we are going to increase roster size of meeting chat and will fix engg code.
```
Comment (JohanFreelancer9) [inline Teams/information-barriers-in-teams.md:178]:
```suggestion
- **Users can't join meetings**: If IB policies are enabled, users are not allowed to join meetings IF the meeting roster size is more than 250 users. The root cause is that IB checks rely on whether users can be added to a meeting chat roster and takes that signal to allow users to join meetings. Joining a meeting once will add that user to the roster, hence for recurring meetings, the roster fills up fast. Once it reaches a count of 250 users, no additional users are allowed to be added to the meeting chat roster. If IB is enabled, users are not allowed to join the meeting, but if IB is not enabled, users are allowed to join the meeting, though they won't be added to the meeting chat roster. A short term solution is to remove inactive members from the meeting chat roster to make space for new users. The size of meeting chat rosters will be increased at a later date and the engg code will be fixed.
```

**Comment by scanum** [comment]
@anwara-ms PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by anwara-ms** [comment]
@scanum I dont think I have write access. How do I merge changes?

**Pull request merged**